package.json

```diff
@@ -48,6 +48,7 @@
         "autocomplete.js": "^0.37.0",
         "brutusin-json-forms": "https://github.com/brutusin/json-forms",
         "deepmerge": "^4.2.2",
+        "fast-deep-equal": "^3.1.3",
```

Comment:
Please add the dependency here: https://github.com/datenanfragen/website/blob/master/ATTRIBUTION